src/zenml/integrations/s3/flavors/s3_artifact_store_flavor.py

```diff
@@ -117,6 +117,8 @@ def _validate_client_kwargs(
         if value is None:
             return value
 
+        value = value.copy()
```

Comment:
why?

Reply:
Was just something I noticed while I was looking at the code + understanding how it worked etc.